pyvo/mivot/viewer/mivot_instance.py

```diff
@@ -128,6 +125,14 @@ def update(self, row, ref=None):
                     setattr(self, self._remove_model_name(key),
                             MivotUtils.cast_type_value(row[ref], getattr(self, 'dmtype')))
 
+    def get_SkyCoord(self):
```

Comment:
Does this have to be a public method, are the users expected to run it?

Reply:
This is the public method the must be run by users. The `SkyCoordBuilder` class is that one that is hidden for the users